Base/Python/tests/test_sitkUtils.py

```diff
@@ -0,0 +1,49 @@
+import slicer
+#import SimpleITK as sitk
+import sitkUtils as su
+
+import unittest
+
+class SitkUtilsTests(unittest.TestCase):
+
+    def runTest(self):
```

Comment:
You need `setUp` and `tearDown`, they are called before and after running each test function.